toolkit/tools/scheduler/schedulerutils/buildlist.go

```diff
@@ -101,8 +102,15 @@ func filterLocalPackagesOnly(packageVersionsInConfig []*pkgjson.PackageVer, inpu
 	return
 }
 
-// ReadReservedFilesList updates the list of reserved files from the manifest file passed in.
+// ReadReservedFilesList updates the list of reserved files (such as toolchain RPMs) from the manifest file passed in.
```

Comment:
Not part of your PR, but since we're here:
```suggestion
// ReadReservedFilesList reads the list of reserved files (such as toolchain RPMs) from the manifest file passed in.
```